Makefile

```diff
@@ -39,7 +39,7 @@ clean: ## Remove generated byte code, coverage reports, and build artifacts
 python-test: clean ## Run tests
 	$(TOX) pip install -r requirements/test.txt --exists-action w
 	$(TOX) coverage run --source="." -m pytest ./eox_core
-	$(TOX) coverage report --fail-under=70
+	$(TOX) coverage report --fail-under=60
```

Comment:
Could you please tell me why you needed to change this?

Reply:
Maybe we are analyzing the covering of something we don't need to cover.

Reply:
Thank you @MaferMazu, you're rigth, we dont need to change this line, the commit is updated, can you please review it?